src/content/blog/working-with-ddev-and-vite.md

```diff
@@ -0,0 +1,482 @@
+---
+title: "Working with Vite and DDEV - an introduction"
+pubDate: 2023-10-28
+modifiedDate: 2023-10-28
+summary: How to work with Vite in DDEV?
+author: Matthias Andrasch
+featureImage:
+  src: /img/blog/2023/08/contributors-working.png
+  alt: Contributors working together on DDEV
+categories:
+  - Community
+---
+
+## Working with DDEV and vite
+
+Vite is a popular web development tool that serves your JavaScript and CSS code in a clever way: Instead of bundling everything like webpack, it uses a technique called "hot module reloading". This enables the ability to instantly update and show your changes in the browser while you're working on your project. 
+
+How can we use Vite within DDEV?
+
+This articles sums up my current personal experience. I hope it will be a helpful  introduction to get started working with Vite and DDEV. 
+
+Vite is written in NodeJS. 
+
+DDEV already has built-in support for [NodeJS](https://ddev.readthedocs.io/en/latest/users/usage/cli/#nodejs-npm-nvm-and-yarn). 
+
+You can set the NodeJS version in `.ddev/config.yaml` for each project ([docs](https://ddev.readthedocs.io/en/latest/users/configuration/config/#nodejs_version)):
+
+```
+nodejs_version: "18"
+```
```

Comment:
Is it important to say this? I would probably suggest removing it, as more-or-less off-topic?

```suggestion
```